tests.py

```diff
@@ -266,6 +290,37 @@ def test_mark_as_read_updates_read(self):
         result = get_json(res)
         self.assertEqual(read, result['read'])
 
+    @single_user
+    def test_update_meta(self):
+        bookmark = {
+            'url': 'https://github.com/',
+        }
+        meta = json.dumps({"blah": True})
```

Comment:
`meta` is now string (encoded json) while should be plain dict. `json.dumps(bookmark)` would handle converting nested structures.

Reply:
While implementation looks okay and should handle both cases just fine, please adjust the test so we know that for sure.